src/main/java/PieceFactory.java

```diff
@@ -11,6 +11,7 @@ public PieceFactory() {}
      * @param type  the type of piece to be created
      * @return      the piece
      */
+    //TODO: add new construct methods to add other parameters
```

Comment:
Not really a TODO, but good to expressedly note this in case a team member decides they wish to further overload `create()`.